tests/frontend/specs/xxauto_reconnect.js

```diff
@@ -11,7 +11,7 @@ describe('Automatic pad reload on Force Reconnect message', function() {
       helper.padChrome$.window.clientVars.automaticReconnectionTimeout = 2;
 
       // open same pad on another iframe, to force userdup error
-      var $otherIframeWithSamePad = $('<iframe src="/p/' + padId + '" style="height: 1px;"></iframe>');
+      var $otherIframeWithSamePad = $('<iframe src="' + padId + '" style="height: 1px;"></iframe>');
```

Comment:
This looks wrong.

Reply:
Take a look at [tests/frontend/helper.js line 128](https://github.com/ether/etherpad-lite/pull/4292/files#r498066819)
also 169, In this case, `padId` is not a logical naming, I have to change it to` padAddress`. Does it make sense to you?

Reply:
Not really ;\

Reply:
Well, the `helper.newPad()` function now on will return pad Address(URL)! which in this case we called it `padId`, and because the `helper.newPad()` generate random nested pad address, i removed the `/p/`.